docs/source/usage/cleaning.rst

```diff
@@ -0,0 +1,186 @@
+Node Cleaning and Maintenance
+=============================
+
+Ironic provides two modes for node cleaning: automated and manual.
+
+Automated Node Cleaning
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Automated cleaning is automatically performed before the first workload has been assigned to a node and when hardware is recycled from one workload to another. Automatic cleaning can be customized for to enable or disable certain features of cleaning.
+
+Common use cases for automated cleaning:
+  * Erase disk drives and destroy all the metadata from previous user.
+  * Debug node failures resulting from disk failures by inspecting the automatic cleaning logs in Ironic conductor.
+  * Cut the time spent in manually wiping and cleaning the node after every workload, saving man hours.
+  * Reset raid configuration to a default state after every workload.
+
+Steps involved in automated cleaning:
+  1. Erase devices: out-of-band disk erase operation on all the supported physical drives in the node. Disk erase cannot be performed on physical drives.
+  2. Erase Devices Metadata: Wipe partition tables, signatures, file-system identifications, and other disk devices.
+  3. RAID Delete (Disabled-by-default): Delete any existing RAID configuration. This step needs to be enabled for automated cleaning.
+  4. Setup RAID configuration in advance(Disabled-by-default): Create RAID configuration from a predefined template when the node in enrolled. This step needs to be enabled for automated cleaning.
+
+
+Enable or disable automated cleaning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable automated cleaning, ensure that your ironic.conf has following option enabled.
+
+.. prompt:: bash $
+
+  [conductor] “ironic.conf”
+  automated_clean=true
+
+To disable it, ensure that your ironic.conf has following option disabled.
+
+.. prompt:: bash $
+
+  [conductor] “ironic.conf”
+  automated_clean=false
+
+
+A cleaning step can be skipped by setting the priority for that cleaning step to zero or ‘None’. Changing the priority of an in-band (ironic-python-agent) cleaning step requires use of a custom HardwareManager, erase_devices is the only exception which can customized from ironic.conf. For instance, to disable erase_devices, you’d set the following configuration option:
```

Comment:
This feels like a new section; maybe "Configuring Automated Cleaning Steps"?

Reply:
For this section, I think it's enough to mention that you enable a step by setting its priority to a number greater than zero; and disable it by setting it to zero. Then have a table with all the possible cleaning steps, a short explanation of what it does, and what its default setting is.

Reply:
Created a new section and removed extra information